pyproject.toml

```diff
@@ -57,6 +57,16 @@ testing = [
     "setuptools",
     "xmlschema",
 ]
+release = [
```

Comment:
I think we should skip this commit ("Move the release extras from tox.ini to pyproject.toml"), because extras are user-visible, and and the release stuff is purely internal.

Reply:
Reverted, I'll clean up before merging.